.github/workflows/regression-tests.yml

```
@@ -175,7 +175,7 @@ jobs:
         make -j $(nproc)
         sudo make install
 
-      # Install Verilator
+      # Install Verilator (Linux)
     - name: Set up Verilator (Ubunutu - source)
```

While we're here editing.
```suggestion
    - name: Set up Verilator (Ubuntu - source)
```